pkgs/top-level/all-packages.nix

```diff
@@ -20289,8 +20289,9 @@ with pkgs;
   texinfo6_5 = callPackage ../development/tools/misc/texinfo/6.5.nix { }; # needed for allegro
   texinfo6_7 = callPackage ../development/tools/misc/texinfo/6.7.nix { }; # needed for gpm, iksemel and fwknop
   texinfo6 = callPackage ../development/tools/misc/texinfo/6.8.nix { };
-  texinfo7 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
-  texinfo = texinfo7;
+  texinfo7_0 = callPackage ../development/tools/misc/texinfo/7.0.nix { };
```

Comment:
Let me ask a naive question: do we even need 7.0 right now?

Reply:
Then I would suggest to use it right away and add a comment when we can switch to 7.1

Reply:
@SuperSandro2000 What are your thoughts around removing `texinfo7_0` in favor of `texinfo7_1` and using the [patch below](https://github.com/NixOS/nixpkgs/pull/262276#issuecomment-1784209023)?

Reply:
Do not remove 7.0 if it is still needed.

Or at least remove it in a separate PR.

Reply:
We can also do this later.